src/server/task-list/controller.js

```diff
@@ -32,9 +34,11 @@ export const taskListGetController = {
 
     const destinationGdsTask = buildGdsTaskItem({
       title: 'Movement destination',
-      initialLink: '#',
-      summaryLink: '#',
-      isValid: destination.validate().isValid,
+      initialLink:
+        destinationValidity.firstInvalidPage?.urlPath ??
+        destination.firstPage.urlPath,
```

Comment:
Hrm - I can see this replicates the pattern we've gone with above, but I *believe* this isn't the pattern we went for.  It was either the firstPage, or it was the summary page.